src/zeynep.js

```diff
@@ -71,11 +71,11 @@
 
       // close subMenu
       subMenuEl.removeClass('opened current')
-      zeynep.find('.submenu.opened:last').addClass('current')
+      zeynep.find('.submenu.opened:last-child').addClass('current')
```

Comment:
Here will cause issues because `last` is a jQuery-specific pseudo and its equivalent is not `last-child` pseudo. Instead it's better to use its equivalent function which is `.last()`.